Forms/ClientUpdaterForm.cs

```diff
@@ -24,22 +25,53 @@ private async void StartUpdate()
         {
             try
             {
+                //If fetch successfully update and load local file.
+                client.Timeout = TimeSpan.FromSeconds(5);
                 string result = await client.GetStringAsync(AppConfig._4RClientsURL);
-                List<ClientDTO> clients = JsonConvert.DeserializeObject<List<ClientDTO>>(result);
-                foreach(ClientDTO clientDTO in clients)
+                LoadServers(result);
+            }catch(Exception)
```

Comment:
```suggestion
            } catch(Exception)
```

Reply:
Maybe we can add as resource file and carry an `fallback-fallback` version. That's right?

Reply:
Just an idea so we can avoid users asking over Discord about how to download this file.

Reply:
Fair enought. I'll do that. 

Thanks for the suggestion.

Reply:
done @afonsocdaniel .